### DISCUSSION TITLE
notification

### DISCUSSION THREAD

**Discussion by Andry042023** [thesamim/TickTickSync#150] [General]
Hello, thank you very much for the plugin!
[
![2024-07-09 17_40_34-Periodic Notes - Plugin Obsidian - PARA - Obsidian v1 6 5](https://github.com/thesamim/TickTickSync/assets/130758297/a8180cb1-cf08-4577-93e1-27e59055a1bf)
![2024-07-09 17_40_41-Periodic Notes - Plugin Obsidian - PARA - Obsidian v1 6 5](https://github.com/thesamim/TickTickSync/assets/130758297/04999979-244a-4f86-b983-85c3a65a0b5a)
](url)
There is a question, is it possible to remove the message that was not explained when I mark any checkbox without the #ticktick tag?

**Comment by Andry042023** [comment]
it would be great to have a palette command to synchronize tick tick manually, so that when you need it you don’t have to go into the settings

**Reply by thesamim** [reply]
@Andry042023 In the next release, there will be a sync button in the ribbon. I will look into also adding a command palette.

**Reply by thesamim** [reply]
> it would be great to have a palette command to synchronize tick tick manually, so that when you need it you don’t have to go into the settings

Command Palette added in 1.1.2
